Library/src/Multiplayer/ErrorCodeStrings.h

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright 2023 Magnopus LLC
+
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#pragma once
+#include <string>
+#include "CSP/Multiplayer/MultiPlayerConnection.h"
+
+namespace csp::multiplayer
+{
+    static std::string ToString(csp::multiplayer::ErrorCode ErrorCode)
```

Comment:
This could be done more efficiently by using the enum value to index into a static string array.

Reply:
I'm also not sure this should be in this namespace if it's just called "ToString". Best to call it "ErrorCodeToString" or make this a generic solution.

Reply:
Also, make it inline :)

Reply:
All great feedback. Thank you both!

Reply:
@MAG-MichaelK if you don't mind, I'll not convert to an array of strings indexed into via the enum, because it would mean we cannot as neatly respond to scenarios where the function is unaware of new error code enum values we introduce in the future.

![image](https://github.com/magnopus-opensource/connected-spaces-platform/assets/99482500/ecd18e5a-eb08-4ab9-a01c-9131d462c3a8)

Since this isn't a hot code path, and since I wouldn't expect the perf characteristics of the switch statement approach to be much of a problem, I'd prefer to keep the function as-is.

Reply:
Most recent commit addresses function name & inline feedback. 👍